documentation/sphinx/source/configuration.rst

```diff
@@ -322,6 +326,23 @@ Backup agent sections
 
 These sections run and configure the backup agent process used for :doc:`point-in-time backups <backups>` of FoundationDB. These don't usually need to be modified. The structure and functionality is similar to the ``[fdbserver]`` and ``[fdbserver.<ID>]`` sections.
 
+.. _configuration-restart-fdbmonitor
+
+Configuring autorestart of fdbmonitor
+=====================================
+
+Configuring the restart parameters for ``fdbmonitor`` is operating system-specific.
+
+In Linux
```

Comment:
If this only applies to our RPM package, we might want to change the header here to be more specific. For example:

```suggestion
Linux (RHEL/CentOS)
```